Sources/BraintreeVenmo/Public/BraintreeVenmo/BTVenmoRequest.h

```diff
@@ -25,17 +27,20 @@ typedef NS_ENUM(NSInteger, BTVenmoPaymentMethodUsage) {
 /**
  * Whether to automatically vault the Venmo account on the client. For client-side vaulting, you must initialize BTAPIClient with a client token that was created with a customer ID. Also, `paymentMethodUsage` on the BTVenmoRequest must be set to `.multiUse`.
  *
- * If this property is set to false, you can still vault the Venmo account on your server, provided that `paymentMethodUsage` is set to `.multiUse`.
+ * If this property is set to false, you can still vault the Venmo account on your server, provided that `paymentMethodUsage` is not set to `.singleUse`.
  *
  * Defaults to false.
  */
 @property (nonatomic) BOOL vault;
 
 /**
- * If set to `.multiUse`, the resulting payment method will be authorized for future payments and can be vaulted. If set to `.singleUse`, the resulting payment method will be authorized for a one-time payment and cannot be vaulted.
+ * If set to `.multiUse`, the Venmo payment will be authorized for future payments and can be vaulted.
+ * If set to `.singleUse`, the Venmo payment will be authorized for a one-time payment and cannot be vaulted.
+ * If set to `.unspecified`, the legacy Venmo UI flow will launch. It is recommended to use `.multiUse` or `.singleUse` for the best customer experience.
  *
- * Defaults to `.multiUse`.
+ * Defaults to `.unspecified`.
  */
+
```

Comment:
```suggestion
```

Reply:
Not sure if this extra space will mess up viewing the doc string in Xcode or the generated Jazzy docs.